test/license-test/run-license-test.sh

```diff
@@ -2,11 +2,11 @@
 set -euo pipefail
 
 SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
-BUILD_BIN="$SCRIPTPATH/../../build"
+BUILD_BIN="$SCRIPTPATH/../../build/bin"
 
-BINARY_NAME="amazon-ec2-metadata-mock"
+BINARY_NAME="amazon-ec2-metadata-mock-linux-amd64"
 LICENSE_TEST_TAG="aemm-license-test"
 
-make -s -f $SCRIPTPATH/../../Makefile build
+make -s -f $SCRIPTPATH/../../Makefile build-binaries
```

Comment:
need to pass in SUPPORTED_PLATFORMS so it doesn't build for all and take forever 

`SUPPORTED_PLATFORMS="linux/amd64 make -s -f $SCRIPTPATH/../../Makefile build-binaries`

Reply:
good call! made the update and verified it only builds for single platform